processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -54,9 +54,14 @@ public ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector)
       return filters.get(0).getBitmapIndex(selector);
     }
 
-    List<ImmutableBitmap> bitmaps = Lists.newArrayList();
-    for (int i = 0; i < filters.size(); i++) {
-      bitmaps.add(filters.get(i).getBitmapIndex(selector));
+    final List<ImmutableBitmap> bitmaps = Lists.newArrayListWithCapacity(filters.size());
```

Comment:
This is more noisy than `new ArrayList<>(filters.size())`. `Lists.newArrayList()`, `Sets.newHashSet()`, etc. are considered obsolete since Java 7: http://stackoverflow.com/a/9980949/648955

I suggest to standardise on `new ArrayList<>()` form and prohibit `Lists.newArrayList()` using checkstyle: http://checkstyle.sourceforge.net/config_regexp.html